demos/smart_framing_demo/cpp_gapi/main.cpp

```diff
@@ -0,0 +1,381 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <chrono>
+#include <string>
+#include <vector>
+
+#include <gflags/gflags.h>
+#include <opencv2/gapi/streaming/cap.hpp>
+#include <opencv2/gapi/imgproc.hpp>
+#include <opencv2/gapi/core.hpp>
+
+#include <opencv2/gapi/fluid/core.hpp>
+#include <opencv2/gapi/fluid/imgproc.hpp>
+#include <opencv2/gapi/cpu/core.hpp>
+#include <opencv2/gapi/cpu/imgproc.hpp>
+#include <opencv2/highgui.hpp>
+
+#include <openvino/openvino.hpp>
+
+#include <monitors/presenter.h>
+#include <utils_gapi/stream_source.hpp>
+#include <utils/args_helper.hpp>
+#include <utils/config_factory.h>
+#include <utils/images_capture.h>
+#include <utils/ocv_common.hpp>
+#include <utils/performance_metrics.hpp>
+#include <utils/default_flags.hpp>
+#include <models/detection_model.h>
+
+
+
+//#include "smart_framing_demo_gapi.hpp"
```

Comment:
??? And demo works without this include. It is weird

Reply:
Understood. Need to remove.

Reply:
Done